$NetBSD: patch-src_apps_deskflow-gui_CMakeLists.txt,v 1.1 2026/01/06 19:57:10 tnn Exp $

CMake Error at src/apps/deskflow-gui/CMakeLists.txt:52 (install):
  install TARGETS RUNTIME_DEPENDENCY_SET is not supported on system "NetBSD"

--- src/apps/deskflow-gui/CMakeLists.txt.orig	2026-01-02 18:59:56.022239122 +0000
+++ src/apps/deskflow-gui/CMakeLists.txt
@@ -51,7 +51,6 @@ target_link_libraries(
 
 install(
   TARGETS ${target}
-  RUNTIME_DEPENDENCY_SET guiDeps
   RUNTIME DESTINATION ${CMAKE_INSTALL_BINDIR}
   BUNDLE DESTINATION .
 )
